infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/util/LinuxInitScriptGenerator.java

```diff
@@ -70,7 +70,12 @@
     }
 
     public String generateNodeDownloadCommand(String rmHostname) {
-        return "wget -nv " + rmHostname + ":8080/rest/node.jar";
+        if (System.getProperty("os.name").contains("Windows")) {
```

Comment:
I agree with Isabelle, this block is useless. 
You can add a separate class 'WindowsScriptGenerator' and call it when the image of the VM uses Windows.

Reply:
Can you validate please? I will check how to do for this scenario in another card